src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-110001.mdx

```diff
@@ -0,0 +1,35 @@
+---
+subject: 'Python agent'
+releaseDate: '2025-10-06'
+version: 11.0.1
+downloadLink: 'https://pypi.python.org/pypi/newrelic'
+features: []
+bugs: ['Prevents early activation in Django Instrumentation', 'Fix server configuration to local configuration sync issues', 'Guard against crashes from trace_cache.pop()']
+---
+
+## Notes
+
+This release of the Python agent provides fixes for early activation in Django instrumentation as well as server side overrides for `max_samples_stored` being ignored.
+
+Install the agent using `easy_install/pip/distribute` via the [Python Package Index](https://pypi.python.org/pypi/newrelic) or download it directly from the [New Relic download site](https://download.newrelic.com/python_agent/release).
+
+## Bug fixes
+
+* Prevents early activation in Django Instrumentation
+
+  * Remove calls to `application_settings()` in Django instrumentation to prevent issues with early activations in pre-fork applications.
```

Comment:
```suggestion
  * Remove calls to `application_settings()` in [Django](https://pypi.org/project/Django/) instrumentation to prevent issues with early activations in pre-fork applications.
```